CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ YodaLocalPackage(yoda-util @yoda/util)
 YodaLocalPackage(yoda-wifi @yoda/wifi)
 YodaLocalPackage(yoda-env @yoda/env)
 YodaLocalPackage(yoda-flora @yoda/flora)
+YodaLocalPackage(yodaos-application @yodaos/application)
```

Comment:
Is this an application framework? I think the yodaos-application looks generic to me.

Reply:
Packages provided officially should be scoped with `@yodaos`.